src/ipmanager/ui/templates/ui/edit_group.html

```diff
@@ -31,8 +31,11 @@ <h2>Editing Group</h2>
 </table>
 </div>
 
-<button type="submit" class="edit">Update Group</button>
-<a href='{% url "single_group" group.key %}'>Cancel</a>
-<a href='{% url "list_all_groups" %}'>List All Groups</a>
+<div style="margin-top: .5em; display: flex; flex-direction: row; gap: .75em; align-items: center;">
```

Comment:
This inline style on the `div` looks like it could be changed to `class="horizontal-container"`.